src/utils.ts

```diff
@@ -163,6 +163,10 @@ export function removeExtension(filename: string): string {
   return filename.replace(/\.(js|mjs|cjs|ts|mts|cts|json|jsx|tsx)$/, "");
 }
 
+export function inferPkgInlinedDeps(pkg: PackageJson): string[] {
+  return [...(pkg.bundledDependencies || [])];
```

Comment:
Npm spec: https://docs.npmjs.com/cli/v11/configuring-npm/package-json#bundledependencies

- It can be either `bundleDependencies` or `bundledDependencies`
- It can be also `true`

But also AFAIK, this field specifies `node_modules` bundled into tarball (which is different than bundling in dist) not sure if this inference is best idea.

Reply:
happy to drop it too - whatever you think

Reply:
lets avoid it for now, explicit is safest imo.